### DISCUSSION TITLE
2021/04/raspberry-kodi/

### DISCUSSION THREAD

**Discussion by giscus** [Li-Aaron/Li-Aaron.github.io#5] [Announcements]
# 2021/04/raspberry-kodi/

本文介绍树莓派安装Kodi，添加IPTV与电影库；并简要介绍相关网络共享设置，以及影片信息刮削。

https://li-aaron.github.io/2021/04/raspberry-kodi/

**Comment by naosense** [comment]
请问你在树莓派上怎么运行tmm？命令行？
